pictures/migrations.py

```diff
@@ -0,0 +1,33 @@
+from django.db.migrations import AlterField
+
+from pictures.models import PictureField
+
+
+class AlterPictureField(AlterField):
+
+    reduces_to_sql = False
+
+    reversible = True
+
+    def database_forwards(self, app_label, schema_editor, from_state, to_state):
```

Comment:
a database_backwards is missing, isn't it?

Reply:
That sadly does not exist. The whole migration API is generally not really extendible.